react/src/blueTheme.ts

```diff
@@ -572,17 +572,48 @@ export const blueTheme: ThemeOptions = {
                 '&$checked + $track': {
                     opacity: 0.38,
                 },
+                '&$checked': {
+                    color: ThemeColors.secondary.main,
+                    '&$disabled': {
+                        color: '#80C3F8',
+                    },
+                    '&$disabled + $track': {
+                        opacity: 0.24,
+                        backgroundColor: ThemeColors.secondary.main,
+                    },
+                },
             },
             colorPrimary: {
+                '&$disabled': {
+                    color: ThemeColors.background.paper,
+                },
+                '&$disabled + $track': {
+                    backgroundColor: PXBColors.black[100],
+                },
                 '&$checked': {
                     color: ThemeColors.primary.main,
+                    '&$disabled': {
+                        color: '#7FBCDF',
+                    },
+                    '&$disabled + $track': {
+                        opacity: 1,
+                        backgroundColor: '#CFE6F3',
+                    },
+                },
+            },
+            colorSecondary: {
+                '&$disabled': {
+                    color: ThemeColors.background.paper,
+                },
+                '&$disabled + $track': {
+                    backgroundColor: PXBColors.black[100],
                 },
             },
             track: {
                 backgroundColor: PXBColors.black[100],
-                opacity: 1,
+                opacity: 0.38,
             },
-            checked: {},
+            disabled: {},
```

Comment:
If we're able to remove `checked` here and still be able to reference it, do we need to have `disabled` here still?